web/text_layer_builder.js

```diff
@@ -40,41 +42,47 @@ import { renderTextLayer } from "pdfjs-lib";
  */
 class TextLayerBuilder {
   constructor({
-    textLayerDiv,
     eventBus,
     pageIndex,
     viewport,
     highlighter = null,
     accessibilityManager = null,
+    isOffscreenCanvasSupported = true,
```

Comment:
Please don't forget to update https://github.com/mozilla/pdf.js/blob/master/web/interfaces.js and https://github.com/mozilla/pdf.js/blob/master/web/default_factory.js accordingly as well.